collectors/VMPropertiesCollector.py

```diff
@@ -19,15 +19,17 @@ def describe(self):
 
     def collect(self):
         g = GaugeMetricFamily('vrops_vm_properties', 'testtest',
-                              labels=['vccluster', 'datacenter', 'virtualmachine', 'hostsystem', 'propkey'])
+                              labels=['vccluster', 'datacenter', 'virtualmachine', 'hostsystem', 'project', 'propkey'])
         i = InfoMetricFamily('vrops_vm', 'testtest',
-                             labels=['vccluster', 'datacenter', 'virtualmachine', 'hostsystem'])
+                             labels=['vccluster', 'datacenter', 'virtualmachine', 'hostsystem', 'project'])
         if os.environ['DEBUG'] >= '1':
             print(self.name, 'starts with collecting the metrics')
+        project_ids = self.get_project_ids_by_target()
 
         thread_list = list()
         for target in self.get_vms_by_target():
-            t = Thread(target=self.do_metrics, args=(target, g, i))
+            pids = project_ids[target]
```

Comment:
rename to `project_ids_target` (everywhere, where pid is used)
Usually I'd refer to a pid as a linux process id.

Reply:
OK, fair enough.